src/context.ts

```diff
@@ -72,8 +72,23 @@ export function commands<C extends Context>() {
 
         ctx.getNearestCommand = (commands, options) => {
             if (ctx.msg?.text) {
-                const userInput = ctx.msg.text.substring(1);
-                return fuzzyMatch(userInput, commands, { ...options });
+                commands = ensureArray(commands);
+                const results = commands.map((commands) => {
+                    const userInput = ctx.msg!.text!.substring(1);
+                    const result = fuzzyMatch(userInput, commands, {
+                        ...options,
+                        language: options?.ignoreLocalization
+                            ? undefined
+                            : ctx.from?.language_code
+                            ? ctx.from.language_code
+                            : undefined,
```

Comment:
**suggestion:** We can clean this up by inverting the condition. This way if `ignoreLocation` is specified and there is a `language_code`, we'll use it. Otherwise `language` will be undefined.

```suggestion
                        language: !options?.ignoreLocalization
                            ? ctx.from?.language_code
                            : undefined,
```